manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Homebrew.Cortexso

```

```suggestion
PackageIdentifier: janhq.HomebrewCortexso
```
Standard practice for GitHub releases is `publisher.repo`